python/ql/test/library-tests/taint/general/Contexts.ql

```diff
@@ -1,9 +1,10 @@
 
 import python
-import semmle.python.security.TaintTest
+import semmle.python.dataflow.Implementation
 import TaintLib
 
 from CallContext context, Scope s
-where exists(CallContext caller | caller.getCallee(_) = context) and context.appliesToScope(s)
+where exists(CallContext caller | caller.getCallee(_) = context) and
+context.appliesToScope(s)
```

Comment:
Can we select the location as well? It's somewhat helpful to be able to link this back to a source location.

Reply:
done